packages/web-components/src/accordion-item/accordion-item.styles.ts

```diff
@@ -0,0 +1,216 @@
+    grid-template-columns: auto auto 1fr auto;
+  }
+
+  .heading-content {
+    height: 100%;
+    display: flex;
+    align-items: center;
+  }
+
+  .button {
+    box-sizing: border-box;
+    appearance: none;
+    border: none;
+    outline: none;
+    text-align: start;
+    cursor: pointer;
+    font-family: inherit;
+    height: 44px;
+    color: ${colorNeutralForeground1};
+    background: ${colorTransparentBackground};
+    line-height: ${lineHeightBase300};
+    height: auto;
+    padding: 0;
+    font-size: inherit;
+    grid-column: auto / span 2;
+    grid-row: 1;
+  }
+
+  .button::before {
+    content: '';
+    position: absolute;
+    inset: 0px;
+    cursor: pointer;
+    border-radius: ${borderRadiusSmall};
+  }
+
+  .icon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    pointer-events: none;
+    position: relative;
+    height: 100%;
+    padding-right: ${spacingHorizontalS};
+    grid-column: 1 / span 1;
+    grid-row: 1;
+  }
+
+  .region {
+    margin: 0 ${spacingHorizontalM};
+  }
+
+  ::slotted([slot='start']),
+  ::slotted([slot='end']) {
+    justify-content: center;
+    align-items: center;
+    padding-right: ${spacingHorizontalS};
+    grid-column: 2 / span 1;
+    grid-row: 1 / span 1;
+  }
+
+  button:focus-visible::after {
+    content: '';
+    position: absolute;
+    inset: 0px;
+    cursor: pointer;
+    border-radius: ${borderRadiusSmall};
+    outline: none;
+    border: 2px solid ${colorStrokeFocus1};
+    box-shadow: inset 0 0 0 1px ${colorStrokeFocus2};
+  }
+
+  /* --- Disabled attr styles --- */
+
+  :host([disabled]) .button {
+    color: ${colorNeutralForegroundDisabled};
+  }
+  :host([disabled]) svg {
+    filter: invert(89%) sepia(0%) saturate(569%) hue-rotate(155deg) brightness(88%) contrast(87%);
+  }
+
+  /* --- Expanded attr styles --- */
+
+  :host([expanded]) .region {
+    display: block;
+  }
+
+  :host([expanded]) .default-collapsed-icon,
+  :host([expanded]) ::slotted([slot='collapsed-icon']),
+  :host(:not([expanded])) .default-expanded-icon,
+  :host(:not([expanded])) ::slotted([slot='expanded-icon']),
+  :host([expanded]) ::slotted([slot='end']),
+  ::slotted([slot='start']),
+  .region {
+    display: none;
+  }
+
+  :host([expanded]) ::slotted([slot='start']),
+  ::slotted([slot='end']) {
+    display: flex;
+  }
+
+  /* --- Appearance attr styles --- */
+
+  .heading {
+    font-size: ${fontSizeBase300};
+    line-height: ${lineHeightBase300};
+  }
+
+  :host([size='small']) .heading {
+    font-size: ${fontSizeBase200};
+    line-height: ${lineHeightBase200};
+  }
+
+  :host([size='large']) .heading {
+    font-size: ${fontSizeBase400};
+    line-height: ${lineHeightBase400};
+  }
+
+  :host([size='extra-large']) .heading {
+    font-size: ${fontSizeBase500};
+    line-height: ${lineHeightBase500};
+  }
+
+  /* --- expandIconPosition attr styles --- */
+
+  :host([expandIconPosition='end']) :slotted(span[slot='start']),
```

Comment:
After changing `expandIconPosition` to kebab case in https://github.com/microsoft/fluentui/commit/faa15937c2189bbf2d671b9c960a0424741e2dbf, you also need to update the selectors in styles.